assembly/guide-configs/errors.json

```diff
@@ -4,10 +4,14 @@
   "file_dir": "references",
   "title": "Errors",
   "sidebar_label": "Errors",
-  "description": "This guide is an overview of the errors that can appear on the Temporal Web UI.",
```

Comment:
There are other errors that appear in the UI, including different history errors and non-history errors like no worker running